lighthouse-core/report/html/templates.html

```diff
@@ -656,11 +666,35 @@
     .lh-category .lh-gauge__label {
       --score-title-font-size: var(--score-title-font-size-big);
       --score-title-line-height: var(--score-title-line-height-big);
+      display: flex;
       margin-top: 14px;
     }
 
+    .lh-gauge__label-wrapper {
+      position: relative;
+      color: var(--body-text-color);
+    }
+
+    .lh-gauge__label-wrapper .tooltip-boundary {
+      display: flex;
+      position: absolute;
+      right: calc(-1 * var(--lh-info-icon-background-size));
+      top: 14px;
+    }
+
+    .lh-gauge__label-wrapper .lh-gauge__label-info-icon {
+      background-image: var(--informative-url);
+      background-size: contain;
+      width: var(--lh-info-icon-background-size);
+      height: var(--lh-info-icon-background-size);
+    }
+
+    .lh-gauge__label-wrapper .tooltip {
+      font-size: var(--caption-font-size);
+      line-height: var(--caption-line-height);
+    }
   </style>
-  <a href="#" class="lh-gauge__wrapper">
+  <a class="lh-gauge__wrapper">
```

Comment:
should these not be `<a>` now?

Reply:
well, they are still links but only in the main gauge header